### PR TITLE
Add sort suggestions

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

**Comment by peternewman** [comment]
Throws:
```
Error: HttpError: Validation Failed: {"resource":"PullRequestReviewComment","code":"custom","field":"pull_request_review_thread.line","message":"pull_request_review_thread.line must be part of the diff"}, {"resource":"PullRequestReviewComment","code":"missing_field","field":"pull_request_review_thread.diff_hunk"}
Error: HttpError: Validation Failed: {"resource":"PullRequestReviewComment","code":"custom","field":"pull_request_review_thread.line","message":"pull_request_review_thread.line must be part of the diff"}, {"resource":"PullRequestReviewComment","code":"missing_field","field":"pull_request_review_thread.diff_hunk"}
```

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -18264,6 +18264,7 @@ jumo->jump
 jumoed->jumped
 jumpimng->jumping
 jumpt->jumped, jump,
+juipter->Jupiter
```
Comment (bdsoha) [inline codespell_lib/data/dictionary.txt:18267]: This is not in sorted order. Please move it to after `"juducial->judicial"`.

```suggestion
```

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:18267]: Yeah that's sort of the point, see https://github.com/codespell-project/codespell/pull/2251#issuecomment-1020349080 and https://github.com/codespell-project/codespell/pull/2251#issuecomment-1022709986

**Comment by DimitriPapadopoulos** [comment]
@peternewman I think the pre-commit addition has made this PR obsolete. Please re-open if that's not the case.

**Comment by peternewman** [comment]
Yeah agreed, and it didn't work properly anyway unfortunately!

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:18267]: This typo is in the dictionary via other means...

**Pull request closed without merging**